org.o20.word.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "org.o20.word",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "word64.run",
+    "tags": ["beta"],
```

Comment:
Flathub in general targets stable builds, or at least ones that are considered good enough for daily use by application developers. I'd delete mentions of "beta", unless you would prefer to merge your app only to `beta` branch for now.